android/app/src/main/res/values-ro/strings.xml

```diff
@@ -782,6 +782,8 @@
 	<string name="pref_tts_open_system_settings">Setări ale sistemului de sinteză vocală</string>
 	<!-- toast displayed when pressing the "Speech synthesis system settings" button, and the system settings aren't found. -->
 	<string name="pref_tts_no_system_tts">Nu au fost găsite setările de sinteză vocală, sunteți sigur că dispozitivul dvs. le acceptă?</string>
+	<string name="drive_through_yes">Comandă din maşină</string>
+	<string name="drive_through_no">Fără drive-through</string>
```

Comment:
Google Maps uses "Servire în mașină" and I guess the negation of that would be "Fără servire în mașină". The language used by e.g. McDonald's and KFC on their Romanian sites is "Drive-thru" so that would work too, paired with "Fără drive-thru".